packages/auth/src/Api/interfaces.ts

```diff
@@ -35,7 +35,7 @@ interface Routes {
   routes: AuthPageRoutes;
 }
 
-export interface AuthState extends Omit<AuthPageProps, 'pageHeader' | 'pageProps'>, Routes {
+export interface AuthState extends Omit<AuthPageProps, 'pageHeader'>, Routes {
```

Comment:
When I leave pageProps in the type, the error occurs
![Screen Shot 2021-02-11 at 10 48 18 AM](https://user-images.githubusercontent.com/54889005/107616910-956f8380-6c57-11eb-82b8-120ae43ae09a.png)